src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -37,24 +37,7 @@ static std::wstring _KeyEventsToText(std::deque<std::unique_ptr<IInputEvent>>& i
 }
 
 #pragma warning(suppress : 26455) // default constructor is throwing, too much effort to rearrange at this time.
-Terminal::Terminal() :
-    _mutableViewport{ Viewport::Empty() },

```

Comment:
This one didn't get copied over to the header. Should we initialize it there too?